packages/components/src/components/modal/modal.jsx

```diff
@@ -35,6 +35,7 @@ const ModalElement = ({
     width,
     renderTitle,
     small,
+    getModalState,
```

Comment:
Please use `onMount` and `onUnmount` function in modal

Reply:
Ok. Will change the implementation